third_party/libosmium/test/t/io/data-cr.opl

```diff
@@ -1 +1 @@
-n1 v1 dV c1 t2014-01-01T00:00:00Z i1 utest T x1.02 y1.02

```

Comment:
This must be reverted

Reply:
Can't do that until will be fixed in master. It create this one automatically with git on any rebase.